techniques/css/C8.html

```diff
@@ -2,83 +2,73 @@
-            content that meets WCAG 2.1.
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。
          </p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>All technologies that support CSS.</p>
-            <p>This technique relates to:</p>
+            <h2>適用 (対象)</h2>
+            <p>CSS に対応しているウェブコンテンツ技術全て</p>
+            <p>これは、次の達成基準に関する達成方法である:</p>
             <ul>
-               <li><a href="https://waic.jp/docs/WCAG21/Understanding/meaningful-sequence">Success Criterion 1.3.2: Meaningful Sequence</a> (Sufficient as a way to meet <a href="../general/G57">G57: Ordering the content in a meaningful sequence</a>)
+               <li><a href="https://waic.jp/docs/WCAG21/Understanding/meaningful-sequence">達成基準 1.3.2: 意味のある順序</a> (<a href="../general/G57">G57: コンテンツを意味のある順序で並べる</a>の達成方法として十分)
                </li>
-               <li><a href="https://waic.jp/docs/WCAG21/Understanding/images-of-text">Success Criterion 1.4.5: Images of Text</a> (Advisory)
+               <li><a href="https://waic.jp/docs/WCAG21/Understanding/images-of-text">達成基準 1.4.5: 文字画像</a> (参考)
                </li>
-               <li><a href="https://waic.jp/docs/WCAG21/Understanding/images-of-text-no-exception">Success Criterion 1.4.9: Images of Text (No Exception)</a> (Advisory)
+               <li><a href="https://waic.jp/docs/WCAG21/Understanding/images-of-text-no-exception">達成基準 1.4.9: 文字画像 (例外なし)</a> (参考)
                </li>
-               <li><a href="https://waic.jp/docs/WCAG21/Understanding/text-spacing">Success Criterion 1.4.12: Text Spacing</a> (Advisory)
+               <li><a href="https://waic.jp/docs/WCAG21/Understanding/text-spacing">達成基準 1.4.12: テキストの間隔</a> (参考)
                </li>
             </ul>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>The objective of this technique is to demonstrate how the visual appearance of spacing
-               in text may be enhanced via style sheets while still maintaining meaningful text sequencing.
-               The CSS letter-spacing property helps developers control the amount of white space
-               between characters. This is recommended over adding blank characters to control the
-               spacing, since the blank characters can change the meaning and pronunciation of the
-               word.
+            <h2>解説</h2>
+            <p>この達成方法の目的は、意味の伝わるテキストの流れを維持しながら、スタイルシートによって表示上の文字間隔を調整する方法を示すことである。 文字間隔の量を調整するには、CSS の letter-spacing プロパティを使用する。 単語の途中に空白文字を入れると意味や発音のされ方が変わってしまうため、間隔の調整はこの方法で行うことが推奨されている。
```

Comment:
text sequencingが「テキストの流れ」は微妙なので…
```suggestion
            <p>この達成方法の目的は、意味のあるテキストのシーケンスを維持しながら、スタイルシートによって表示上の文字間隔を調整する方法を示すことである。文字間隔の量を調整するには、CSS の letter-spacing プロパティを使用する。単語の途中に空白文字を入れると意味や発音のされ方が変わってしまうため、間隔の調整はこの方法で行うことが推奨されている。
```